rpcs3/Emu/System.cpp

```diff
@@ -1188,7 +1192,7 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 			if (!bdvd_dir.empty() && !fs::is_file(bdvd_dir + "PS3_DISC.SFB"))
 			{
 				// Unuse if invalid
-				sys_log.error("Failed to use custom BDVD directory: '%s'", bdvd_dir);
+				sys_log.notice("Ignoring empty vfs BDVD directory: '%s'", bdvd_dir);
```

Comment:
If the user has tried to set a custom BDVD image he would like to be notified in a noticeable way if it doesn't work.

Reply:
I think I'll just remove this turd of code.
I don't see any reason why any user would do that

Reply:
updated. it will now only display the error if the directory is not empty.